src/app/containers/RadioSchedule/index.jsx

```diff
@@ -49,17 +49,23 @@ const RadioScheduleContainer = ({ endpoint }) => {
       propSatisfies(time => time < currentTime, 'publishedTimeStart'),
     )(radioScheduleData.schedules);
 
-    console.log(latestProgrammeIndex);
-    
     const schedulesToShow = radioScheduleData.schedules.slice(
       latestProgrammeIndex - 2,
       latestProgrammeIndex + 2,
     );
-    const latestProgram = radioScheduleData.schedules[latestProgrammeIndex];
-
-    console.log('current time', new Date());
 
-    console.log('current time from var', new Date(currentTime));
+    const latestProgram = radioScheduleData.schedules[latestProgrammeIndex];
+    if (
```

Comment:
Aha, I think this is slightly the incorrect order which is making this a little more tricky than it needs to be! So we want to show the live _or_ latest on demand programme in the first position at all times, and then go chronologically back for the next two - which means if we take call the 'most recent' programme _n_, we want to show _n_, _n-1_, _n-2_, _n+1_. It might be easiest to then create a new array here, instead of slicing the schedules data above and trying to manipulate it like this - you could do something like this for example:

```
const { schedules } = radioScheduleData;
const schedulestoShow = [
  schedules[latestProgrammeIndex],
  schedules[latestProgrammeIndex - 1],
  schedules[latestProgrammeIndex - 2],
  schedules[latestProgrammeIndex + 1],
];
```

Let me know if you have any questions on this!

Reply:
Sorry, I misunderstood the order. Have updated the PR with these changes now. The only issue was that unlike in real ares data, right now we don't always have a "NEXT" program available for services. And this caused radio schedule not to get rendered in storybook because it could not access `schedules[latestProgrammeIndex + 1]`. So for now I have updated data jsons to include a "NEXT" program.

Didn't seem like adding logic for checking if "NEXT" program is available in the data is necessary because this would not happen with actual ares data. However, if you feel otherwise, let me know!

Reply:
I think we should have enough logic to ensure we don't break the page if ares fails or gives us the wrong data, but just not render the container at all if we don't have the expected number of items rather than try and fallback in a clever way. The issue currently in progress to fix time machine should mean that we can set the server time to a known value when we're testing this against fixture data :)